content/applications/general/email_communication/email_servers.rst

```diff
@@ -129,94 +129,80 @@ How to manage inbound messages
 
```

Comment:
I couldn't comment on the line that had the header title `How to manage inbound messages`, but please

- Delete `How to`
- Capitalize the `m` in `manage`